migrations_sqlite/2023-09-07-131701_create_db/up.sql

```diff
@@ -0,0 +1,24 @@
+-- Your SQL goes here
+
+CREATE TABLE ownership_voucher (
+    guid varchar(36) NOT NULL PRIMARY KEY,
+    contents blob NOT NULL
+);
+
+CREATE TABLE manufacturer_vouchers (
+    ov_guid varchar(36) NOT NULL PRIMARY KEY, 
+    FOREIGN KEY (ov_guid) REFERENCES ownership_voucher(guid) ON DELETE CASCADE
```

Comment:
should the same field be both primary and foreign key?

Reply:
yes